src/components/AssetItem/assetItem.scss

```diff
@@ -0,0 +1,99 @@
+.asset-item {
+  all: initial;
+
+  @include box;
+  @include font;
+  @include flex-row;
+
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 20px 0;
+  height: 80px;
+  cursor: pointer;
+
+  > .info {
+    @include flex-row;
+
+    align-items: center;
+
+    > .symbol {
+      @include flex-row;
+
+      width: 80px;
+      justify-content: center;
+
+      &.-icon {
+        opacity: 0.2;
+      }
+
+      &.-text {
+        height: 36px;
+        padding-top: 11px;
```

Comment:
not so good use paddings for centring of elements. In this case you can use `align-items: center` for vertical alignment